tests/conf.py

```diff
@@ -0,0 +1,69 @@
+import random
+from datetime import datetime, timedelta
+
+from calendar_app.models import Status, Task
+
+example_task_list = [
```

Comment:
this list can be generated based on json file, the json file with tasks is more flexible